ui/admin/app/components/roles/role/actions/index.hbs

```diff
@@ -3,38 +3,21 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-{{!TODO Refactor page to use hds dropdown in ICU-13879}}
-<Rose::Dropdown
-  @text={{t 'actions.manage'}}
-  @dropdownRight={{true}}
-  as |dropdown|
->
+<Hds::Dropdown as |dd|>
+  <dd.ToggleButton @text= {{t 'actions.manage'}} @color ='secondary'/>
```

Comment:
```suggestion
  <dd.ToggleButton @text={{t 'actions.manage'}} @color='secondary'/>
```
Did you commit this code via vscode? We have a linter that runs when `git commit` is run but I wonder if it doesn't run when vscode is used.

Reply:
Yea, I did commit it thru vscode, but would it run if I did `git commit` thru the command line?

Reply:
it will run the linter if you run `git commit` via the command line. I actually will use vscode to stage changes but I always use the command line to commit so the different commit checks are run.